src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -152,12 +145,6 @@ public Hardware() {
             intakeMotor2 = new WPI_TalonFX(INTAKE_OUTER_MOTOR);
             intakeSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, INTAKE_SOLENOID_UP,
                     INTAKE_SOLENOID_DOWN);
-            if (I2C_MUX_ENABLED) {
```

Comment:
In `Hardware.java`, `I2C_MUX_ENABLED` can be removed as well (github doesn't let me comment there).